libs/java/dal/db/.devcontainer/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/
 
 # [Optional] Uncomment this section to install additional OS packages.
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get -y install --no-install-recommends build-essential dnsutils
+    && apt-get -y install --no-install-recommends build-essential dnsutils gnupg2
```

Comment:
not certain if this is needed, but it's supposed to provide a way to link your keys to the OS host.